client/src/hooks/useAuth.tsx

```diff
@@ -0,0 +1,102 @@
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useState,
+  useCallback,
+} from "react";
+
+export const BASE_API_URL = import.meta.env.VITE_API_URL;
+
+interface User {
+  id: string;
+  handle: string;
+  avatar: string;
```

Comment:
Changes to this file are unnecessary for this PR. Also change the filename back to `useAuth.ts`.